src/App.css

```
@@ -5,10 +5,47 @@
+  --color-faint: #9b978f; /* years, tags, company taglines */
+  --color-hint-line: #bdb9b0; /* scroll-hint vertical line */
+  --color-tag-hover-border: rgba(0, 0, 0, 0.35);
+  --color-tag-hover-bg: rgba(0, 0, 0, 0.04);
+  --color-code-bg: rgba(0, 0, 0, 0.06);
+  --color-pre-bg: rgba(0, 0, 0, 0.04);
+  --color-cursor-bg: #000;
+  --color-cursor-text: #fff;
   --font-body: 'IBM Plex Serif', serif;
   --font-heading: 'IBM Plex Serif', serif;
   --content-max: 1120px;
   max-width: var(--content-max);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)